.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: Broken or unintended behavior with the library.
+labels: 'bug-unconfirmed'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Before Submitting your Issue
+        Our GitHub issues are meant exclusively for bug reports and feature requests. For questions about using MSAL for iOS and macOS, please refer to the [common issues and FAQ](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki#getting-help-common-issues-and-faq). If your question is not addressed there, we encourage you to utilize Chat with Copilot for further assistance.
+        
+        For issues affecting your app in production, please create a support ticket with your Microsoft representative.
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Issue Details
+
+  - type: input
+    attributes:
+      label: MSAL Version
+      description: "Please enter the latest MSAL version where the issue can be reproduced."
+      placeholder: "1.3.0"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: "Briefly describe the issue."
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Error Message
+      description: "Provide any error message or stack trace you're encountering."
+
+  - type: textarea
+    attributes:
+      label: MSAL Logs
+      description: "Share [verbose or trace level log messages](https://learn.microsoft.com/en-us/entra/msal/objc/logging-ios?tabs=objc#logging-levels) from MSAL. For detailed instructions on collecting MSAL logs, refer to our [Logging Guide](https://learn.microsoft.com/en-us/entra/msal/objc/logging-ios?tabs=objc)."
+
+  - type: textarea
+    attributes:
+      label: Reproduction Steps
+      description: "Provide steps to reproduce the issue or link to a sample demonstrating this behavior."
```

Comment:
I don't think we should be asking for a link. They can attach it to their issue

Reply:
Updated